File: app/src/main/java/org/sco/movieratings/presenter/BottomBarPresenter.java
package org.sco.movieratings.presenter;

import android.view.View;

import com.google.android.material.bottomnavigation.BottomNavigationView;

import org.sco.movieratings.R;

import androidx.annotation.NonNull;

public class BottomBarPresenter {

    private BottomNavigationView mBottomNavigationView;

    public BottomBarPresenter(@NonNull View view) {
        mBottomNavigationView = view.findViewById(R.id.bottom_navigation);
            }

    public void setSelectedItemById(int itemId) {
        mBottomNavigationView.setSelectedItemId(itemId);
    }

    public void setOnNavigationItemSelectedListener(
            BottomNavigationView.OnNavigationItemSelectedListener listener) {
        mBottomNavigationView.setOnNavigationItemSelectedListener(listener);
    }
}

File: app/src/main/java/org/sco/movieratings/presenter/BottomBarPresenter.kt
package org.sco.movieratings.presenter

import android.view.View
import com.google.android.material.bottomnavigation.BottomNavigationView
import org.sco.movieratings.R

class BottomBarPresenter(val view: View) {

    private val bottomNavigationView: BottomNavigationView by lazy {
        view.findViewById<BottomNavigationView>(R.id.bottom_navigation)
    }

    fun setSelectedItemById(itemId: Int) {
        bottomNavigationView.selectedItemId = itemId
    }

    fun setOnNavigationItemSelectedListener(listener: BottomNavigationView.OnNavigationItemSelectedListener) {
        bottomNavigationView.setOnNavigationItemSelectedListener(listener)
    }
}